code/modules/mob/living/silicon/pai/software/pai_apps.dm

```diff
@@ -317,16 +317,17 @@
 					to_chat(usr, "<span class='warning'>You are already hacking that door!</span>")
 				else
 					hacking = TRUE
-					INVOKE_ASYNC(src, /datum/pai_software/door_jack/.proc/hackloop)
+					INVOKE_ASYNC(src, .proc/hackloop)
 		if("cancel")
 			hackdoor = null
 		if("cable")
-			if(cable)
-				to_chat(usr, "<span class='warning'>You already have a cable deployed!</span>")
-				return
-			var/turf/T = get_turf(pai_holder)
-			cable = new /obj/item/pai_cable(T)
-			pai_holder.visible_message("<span class='warning'>A port on [pai_holder] opens to reveal [cable], which promptly falls to the floor.</span>")
+			playsound(pai_holder, 'sound/mecha/mechmove03.ogg', 25, TRUE)
```

Comment:
I agree. Please remove the sound. The AI is already notified of these hacks, and it takes a good while to complete.

Reply:
As stated above, the AI is *not* notified about the hacks, and hasn't been since #13346 was merged. The hack itself takes an average of 10 seconds to complete, and is currently completely silent except for the sound of the door opening.
If someone really wanted to be stealthy about it even with the sound, they could just extend the cable somewhere else and walk over.